app/models/post.rb

```diff
@@ -0,0 +1,16 @@
+class Post < ApplicationRecord
+  has_many :likes
+  has_many :comments
+  belongs_to :user, foreign_key: true
+  after_save :update_posts_counter
+
+  def check_last_five_posts
+    comments.last(5)
+  end
+
+  private
+
+  def update_posts_counter
+    user.update(posts_counter: user.posts.length)
+  end
```

Comment:
- [ ] [OPTIONAL] Basically this method is supposed to increment the counter for the posts for a specific user. This can easily be achieved by making use of the ActiveRecord increment!. Kindly consider using this. Have a look at [this](https://www.rubydoc.info/docs/rails/ActiveRecord%2FPersistence:increment!) or [this](https://stackoverflow.com/questions/11499110/increment-vs-1) for inspiration. 

**You will see how shorter this line will become** :smile:

Reply:
It helped me a lot actually cuz I was struggling a little bit with that functionality, thank you so much, now it works perfectly.